core-skills/web-application-development-with-react/README.md

```diff
@@ -0,0 +1,74 @@
+# Web Application Development with React
+
+## Summary
+
+1. Can describe situations where you should/shouldn't decide to use React for a project
+2. Can setup a React project from scratch in a repeatable way
+3. Can describe the different parts of the Javascript ecosystem (e.g. NPM, Webpack, etc)
+4. Can demonstrate using TDD on business logic within a React code base
+5. Can demonstrate strategies to achieve clear separation of business logic and display logic
+with React components
+6. Can demonstrate applying sensible styling to React components
+7. Can demonstrate strategies and tools to run tests against an API within a React application
+
+## Sections
+
+1. [React - Why and when should I use it?](./why-and-when.md)
+2. [Setting up a React project](./setup.md)
+3. [React Fundamentals](./fundamentals/README.md)
+4. [Prototyping React Components](./prototyping.md)
+5. [Styling React Components](./prototyping.md)
+
+## Mark Scheme
+
+### Hedgehog
+
+Learning the fundamentals - How not to hurt yourself with React
+
+- [ ] Can demonstrate an understanding of the benefits of using React
```

Comment:
Would like to see more specific `required` criteria on these, as we have on the TDD mark scheme/secret reviewers guide.

Reply:
I thought that, but the `why-and-when.md` file does expain things.